template/build/webpack.prod.conf.js

```diff
@@ -1,4 +1,5 @@
 var path = require('path')
+var glob = require('glob-all')
```

Comment:
I think since you've been able to combine the glob bellow into a single pattern we no longer need `glob-all`, we can just use node's built in `glob` right? :)